README.md

```diff
@@ -29,7 +29,7 @@ access the video decoding features available on your GPU.
 > [!IMPORTANT]
 > `sudo amdgpu-install --usecase=rocm`
 
-* [Video Acceleration API](https://en.wikipedia.org/wiki/Video_Acceleration_API) - `Libva` is an implementation for VA-API
+* [Video Acceleration API](https://en.wikipedia.org/wiki/Video_Acceleration_API) - `libva-amdgpu-dev` is an AMD implementation for VA-API
```

Comment:
The prerequisites.rst doesn't have libva-amdgpu-dev mentioned specifically. Is that something that needs to be added?

Reply:
Updated